datadog/threadstats/base.py

```diff
@@ -116,6 +122,13 @@ def event(self, title, text, alert_type=None, aggregation_key=None,
             'The web server is up again', alert_type='success')
         """
         if not self._disabled:
+            # Append all client level tags to every event
+            if self.constant_tags:
+                if tags:
+                    tags += self.constant_tags
+                else:
+                    tags = self.constant_tags
+
```

Comment:
Or I can add to dogstatsd. We have a use case for it.

On Fri, Oct 16, 2015, 9:12 AM Yann <notifications@github.com> wrote:

> In datadog/threadstats/base.py
> <https://github.com/DataDog/datadogpy/pull/90#discussion_r42253553>:
>
> > @@ -116,6 +122,13 @@ def event(self, title, text, alert_type=None, aggregation_key=None,
> >              'The web server is up again', alert_type='success')
> >          """
> >          if not self._disabled:
> > +            # Append all client level tags to every event
> > +            if self.constant_tags:
> > +                if tags:
> > +                    tags += self.constant_tags
> > +                else:
> > +                    tags = self.constant_tags
> > +
>
> constant_tags are not associated to events in dogstatsd. To be
> consistent, we should do the same with ThreadStats, what do you think ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DataDog/datadogpy/pull/90/files#r42253553>.
>